src/components/Dissolution/CompleteResolution.vue

```diff
@@ -725,6 +730,7 @@ ul {
 // were not using the expected color of $gray7
 ::v-deep label.v-label.theme--light {
   color: $gray7 !important;
+  animation: none !important;
```

Comment:
Your PR description says you removed this from both pages but I only see it on this page...?

Also, check the Vuetify docs as there's probably a prop you can pass to the component to disable animations.

Reply:
Are you saying this CSS override is affecting the other component too?

Reply:
I haven't been able to find any props that disable animations. in the v-checkbox component.

Reply:
There are "base Vuetify props" that aren't listed for each component. Either Google it or try it (might be called "transition").

I'm pushing a bit for this because overriding CSS to kill an animation is not the right way to do it.

Reply:
> Are you saying this CSS override is affecting the other component too?

Yes they are actually both created in this file and then only one is displayed depending on the filing type.

Okay I'll do some more reading tonight and get it figured!

Reply:
Okay I finally got it. It was a rule prop in the v-checkbox I spend way to much time looking in the wrong area. It should be good now. I reapplied the animation to the complete resolution component. As it is very clearly an intentional addition now that I know how the component is working.